src/core/mixins/handlerWallet.mixin.js

```diff
@@ -0,0 +1,94 @@
+import utils from 'web3-utils';
+import {
+  getEthBalance,
+  subscribeToAccountBalance,
+  getUSDPrice
+} from '@/apollo/queries/wallets/wallets.graphql';
+import { Toast, ERROR } from '@/modules/toast/handler/handlerToast';
+
+const tokens = {
+  eth: 'ethereum'
+};
+
+export default {
+  name: 'HandlerWallet',
+  data() {
+    return {
+      getEthBalance: '',
+      getLatestPrices: '',
+      subscribeToAccountBalance: ''
+    };
+  },
+  apollo: {
+    /**
+     * Apollo query to get eth balance
+     */
+    getEthBalance: {
+      query: getEthBalance,
+      variables() {
+        return {
+          hash: this.address
+        };
+      },
+      skip() {
+        return !this.isEthNetwork;
+      },
+      result({ data }) {
+        this.setAccountBalance(utils.toBN(data.getEthBalance.balance));
+      },
+      error(error) {
+        Toast(error.message, {}, ERROR);
+      }
+    },
+    /**
+     * Apollo subscription for eth balance
+     */
+    $subscribe: {
+      subscribeToAccountBalance: {
+        query: subscribeToAccountBalance,
+        variables() {
+          return {
+            owner: this.address,
+            event: 'NEW_ETH_TRANSFER'
```

Comment:
you have access to this enum, you can export this

Reply:
added